Error when adding options to checkbox / radio question by pressing enter

**Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue. Please don't comment if you have no relevant information to add!**

**Describe the bug**
When adding an option to a checkbox / radio question, I often get "Error while saving the answer" if I type in the option and then press enter. If I press the plus button it always works. The option is added to the form though (and is still visible after reloading).

It seems to send a patch request to update a non-existent option when this happens, after sending the post request to create the option.

After this bug happens, I then get an error "There was an issue deleting this option" when clicking anywhere else on the page.

This is a new problem since version 5.2.5.

**To Reproduce**

1. Create a new form
2. Add a checkbox / radio question
3. Type in an option and press enter
4. See error "Error while saving the answer"
5. Click anywhere else
6. See error "There was an issue deleting this option"

**Expected behavior**
I expect to be able to add an option by pressing enter.

**Screenshots**

<img width="1796" height="1068" alt="Image" src="https://github.com/user-attachments/assets/f1724bd9-5b7c-44b9-b7ef-ca913c903a6c" />

**Nextcloud (please complete the following information):**

- Nextcloud-Version: 33.0.0
- Forms-Version: 5.2.5

**Desktop (please complete the following information):**

- OS: Linux
- Browser: Firefox
- Version 148.0

**Browser log**

<img width="1250" height="1270" alt="Image" src="https://github.com/user-attachments/assets/e5c5e7bc-00cf-41af-a387-e1801bdce21d" />

<img width="713" height="97" alt="Image" src="https://github.com/user-attachments/assets/2fd7537d-575c-4c9a-b661-18a4d7fde962" />

When creating the option with enter:
```
XHRPATCH
https://cloud.rsadelft.nl/ocs/v2.php/apps/forms/api/v3/forms/137/questions/647/options/{optionId}
[HTTP/2 404  155ms]

[ERROR] forms: Error while saving answer 
Object { app: "forms", uid: "anitters", level: 2, answer: {…}, error: AxiosError }
​
answer: Object { local: Getter & Setter, questionId: Getter & Setter, text: Getter & Setter, … }
​​
__ob__: Object { shallow: false, mock: false, vmCount: 0, … }
​​
local: false
​​
order: 7
​​
questionId: 649
​​
text: ""
​​
<get local()>: function get()
​​
<set local()>: function set(m)
​​
<get order()>: function get()
​​
<set order()>: function set(m)
​​
<get questionId()>: function get()
​​
<set questionId()>: function set(m)
​​
<get text()>: function get()
​​
<set text()>: function set(m)
​​
<prototype>: Object { … }
​
app: "forms"
​
error: AxiosError: Request failed with status code 404
​
level: 2
​
uid: "anitters"
​
<prototype>: Object { … }
index.mjs:47:17
    log index.mjs:47
    error index.mjs:65
    updateAnswer AnswerInput.vue:396
    onInput AnswerInput.vue:241
    debounceOnInput AnswerInput.vue:209
    add index.js:378
    #L index.js:252
    add index.js:416
    add index.js:352
    debounceOnInput AnswerInput.vue:209
    o index.js:27
    h index.js:40
    (Async: setTimeout handler)
    h index.js:35
    (Async: setTimeout handler)
    m index.js:61
    VueJS 33
```

Then when clicking anywhere else:
```
XHRDELETE
https://cloud.rsadelft.nl/ocs/v2.php/apps/forms/api/v3/forms/138/questions/649/options/{optionId}
[HTTP/2 400  60ms]

[ERROR] forms: Error while deleting an option 
Object { app: "forms", uid: "anitters", level: 2, error: AxiosError, option: {…} }
​
app: "forms"
​
error: AxiosError: Request failed with status code 400
​
level: 2
​
option: Object { local: Getter & Setter, questionId: Getter & Setter, text: Getter & Setter, … }
​​
__ob__: Object { shallow: false, mock: false, vmCount: 0, … }
​​
local: false
​​
order: 7
​​
questionId: 649
​​
text: ""
​​
<get local()>: function get()
​​
<set local()>: function set(m)
​​
<get order()>: function get()
​​
<set order()>: function set(m)
​​
<get questionId()>: function get()
​​
<set questionId()>: function set(m)
​​
<get text()>: function get()
​​
<set text()>: function set(m)
​​
<prototype>: Object { … }
​
uid: "anitters"
​
<prototype>: Object { … }
index.mjs:47:17
    log index.mjs:47
    error index.mjs:65
    deleteOptionFromDatabase QuestionMultipleMixin.ts:227
    (Async: promise callback)
    deleteOptionFromDatabase QuestionMultipleMixin.ts:226
    deleteOption QuestionMultipleMixin.ts:189
    checkValidOption QuestionMultipleMixin.ts:173
    checkValidOption QuestionMultipleMixin.ts:171
    VueJS 4
    handleTabbing AnswerInput.vue:215
    VueJS 3
    focus AnswerInput.vue:222
    focusIndex QuestionMultipleMixin.ts:112
    updateAnswer QuestionMultipleMixin.ts:159
    VueJS 5
    updateAnswer QuestionMultipleMixin.ts:159
    VueJS 5
    updateAnswer QuestionMultipleMixin.ts:158
    VueJS 4
    onInput AnswerInput.vue:246
    debounceOnInput AnswerInput.vue:209
    add index.js:378
    #L index.js:252
    add index.js:416
    add index.js:352
    debounceOnInput AnswerInput.vue:209
    o index.js:27
    h index.js:40
    (Async: setTimeout handler)
    h index.js:35
    (Async: setTimeout handler)
    m index.js:61
    VueJS 33
```

**Additional context**

--- a/src/components/Questions/AnswerInput.vue
+++ b/src/components/Questions/AnswerInput.vue
@@ -347,16 +347,24 @@ export default {
 			const answer = { ...this.answer }
 			answer.text = value
 
-			// Dispatched for creation. Marked as synced
-			this.$set(this.answer, 'local', false)
-			const newAnswer = await this.createAnswer(answer)
+			// Prevent any queued debounced PATCHes from running while creating
+			this.queue.pause()
+			try {
+				// Dispatched for creation. Marked as synced
+				this.$set(this.answer, 'local', false)
+				const newAnswer = await this.createAnswer(answer)
 
-			// Forward changes, but use current answer.text to avoid erasing
-			// any in-between changes while creating the answer
-			newAnswer.text = this.$refs.input.value
-			this.localText = ''
+				// Forward changes, but use current answer.text to avoid erasing
+				// any in-between changes while creating the answer
+				newAnswer.text = this.$refs.input.value
+				this.localText = ''
 
-			this.$emit('create-answer', this.index, newAnswer)
+				this.$emit('create-answer', this.index, newAnswer)
+			} finally {
+				// Clear pending update tasks (stale PATCHes) before resuming processing
+				this.queue.clear()
+				this.queue.start()
+			}
 		},
 
 		/**

